src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1187,9 +1187,11 @@ Prevent Long URL From Breaking Out of Container
 /*
 Glyphicon for success status messages
 */
-.alert-success:before {
+
+.icon-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 2px;
+    padding-right: .5em;
     content: '\e084';
+    vertical-align: middle;
```

Comment:
After playing around with it and discussing with @LiHaoTan, it seems that `top` and `bottom` (both show identical behaviour in this case) result in better vertical alignment of the icon with the text. This is probably due to the peculiarities of the Glyphicons Halflings font when used together with the Helvetica title text. After examining the original `glyphicon` class and finding out that it simply hardcodes a `1px` spacing from the top via relative positioning such that the icon happens to vertically align with the text, our conclusion is to use
```css
vertical-align: top;
```
for its simplicity (it also happens to vertically middle align the icon to the text).